.circleci/docker/Dockerfile

```diff
@@ -37,38 +37,15 @@ ENV CXX "${cxx}"
 # Whether to build for TPUVM mode
 ENV TPUVM_MODE "${tpuvm}"
 
-# Rotate nvidia repo public key (last updated: 04/27/2022)
```

Comment:
I wonder why we remove this part (line40-line52)

Reply:
I don't think any CUDA related packages need to be installed anymore, as base image should already have everything to build ptxla with cuda support

Reply:
ah that makes sense. Thanks!